packages/webview_flutter/lib/src/tizen_webview.dart

```diff
@@ -33,8 +29,10 @@ class TizenWebView {
   Future<bool?> _onMethodCall(MethodCall call) async {
     switch (call.method) {
       case 'javaScriptChannelMessage':
-        final String channel = call.arguments['channel']! as String;
-        final String message = call.arguments['message']! as String;
+        final Map<String, Object?> arguments =
+            (call.arguments as Map<Object?, Object?>).cast<String, Object?>();
+        final String channel = arguments['channel']! as String;
+        final String message = arguments['message']! as String;
```

Comment:
I have a question.
I understand it uses `Map` because of `avoid_dynamic_calls`.
Is cast even necessary in this situations that the type is predictable as String?
I just wonder if it is possible to do like below.
```dart
final Map<String, String?> arguments =
    call.arguments as Map<String, String?>;
final String channel = arguments['channel']!;
final String message = arguments['message']!;
```

Reply:
No, that's not possible. `as` cannot replace `Map.cast()` since they are different things. An object of type `dynamic` or `Object?` must first be converted to either a `Map<dynamic, dynamic>` or `Map<Object?, Object?>` (the latter is preferred) using `as`, and then casted to a specific type (`Map<K, V>`) using either `Map.cast()` or `Map.from()` (the former is preferred because it involves less copying).

Reply:
FYI) The code can be simplified to

```dart
final arguments = (call.arguments as Map).cast<String, Object?>();
```

if you disable the `always_specify_types` lint rule. By default it's disabled in the core lints, but we use strict typing with custom analysis options in this repo.

Reply:
Oh.. OK. Thank you for detail reply. It helped me understand :) .